girder/constants.py

```diff
@@ -195,6 +195,12 @@ class TokenScope:
     ANONYMOUS_SESSION = 'core.anonymous_session'
     USER_AUTH = 'core.user_auth'
     TEMPORARY_USER_AUTH = 'core.user_auth.temporary'
+    PLUGINS_ENABLED_READ = 'core.plugins.read'
```

Comment:
Like I said in an earlier post, this list is currently put forth as a straw man. The actual decisions about which scopes should exist should be debated by the community at large.